recipes/busybox/busybox-1.24.2/default_defconfig

```diff
@@ -0,0 +1,1073 @@
+#
+# Automatically generated make config: don't edit
+# Busybox version: 1.24.1
```

Comment:
Nit picking contest accepted ;)
The comment is correct, it was automatically generated with busybox version 1.24.1.
Updating the comment would therefore be falsifying the evidence.